polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -84,14 +88,52 @@ bool isCandidate(LoopLikeOpInterface loop) {
   return true;
 }
 
+unsigned getLocalMemRemain(Operation *module) {
```

Comment:
[Suggestion]: pass a reference to avoid having to check for nullptr.

Reply:
done.